Client/Frontend/PasswordManagement/DevicePasscodeRequiredViewController.swift

```diff
@@ -9,7 +9,7 @@ class DevicePasscodeRequiredViewController: SettingsViewController {
     private var warningLabel: UILabel = {
         let label = UILabel()
         label.translatesAutoresizingMaskIntoConstraints = false
-        label.font = LegacyDynamicFontHelper().DeviceFontExtraLarge
+        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body, size: 17)
```

Comment:
Looks like I wrote the wrong font size in the ticket and it should actually be 19. Sorry about that! Could you please change it everywhere?

Reply:
@thatswinnie  , Are there any other changes required other than font size ?

Reply:
@akshat3358 thanks for making this change. The only other change necessary would be to add `import Common` to `DevicePasscodeRequiredViewController.swift` and `PasswordManagerOnboardingViewController.swift` like mentioned above.